runtime/sema/authaccount_type.go

```diff
@@ -690,6 +703,36 @@ var AccountKeysTypeGetFunctionType = &FunctionType{
 	RequiredArgumentCount: RequiredArgumentCount(1),
 }
 
+// fun forEach(_ function: (AccountKey: Bool)): Void
+var AccountKeysTypeForEachFunctionType = func() *FunctionType {
+	// fun (_ key: AccountKey): Bool
+	iterFunctionType := &FunctionType{
+		Parameters: []*Parameter{
+			{
+				Identifier:     "accountKey",
```

Comment:
Maybe just remove
```suggestion
```

Reply:
manually writing rank-2 functions as an AST is confusing